src/Conversion/KrnlToLLVM/ConvertKrnlToLLVM.hpp

```diff
@@ -0,0 +1,85 @@
+
+/*
+ * SPDX-License-Identifier: Apache-2.0
+ */
+
+//====------ ConvertKrnlToLLVM.hpp - Krnl Dialect Lowering  ---------------===//
+//
+// Copyright 2019-2022 The IBM Research Authors.
+//
+// =============================================================================
+//
+// Lowering of Krnl operations to a combination of other dialects.
+//
+//===----------------------------------------------------------------------===//
+
+#pragma once
+
+#include "src/Conversion/KrnlToLLVM/ConvertKrnlToLLVM.hpp"
+#include "src/Dialect/Krnl/KrnlOps.hpp"
+#include "src/Pass/Passes.hpp"
+#include "src/Support/Common.hpp"
+
+const std::string DEFAULT_DYN_ENTRY_POINT = "run_main_graph";
+
+using namespace mlir;
```

Comment:
It's generally a good idea to avoid `using` statements for namespaces in headers.

Reply:
Sorry @sstamenova I merged the PR already before seeing this comment. I will post another PR and remove the using declaration in the header file.

Reply:
PR posted.